examples/mig_with_percent/simple/README.md

```diff
@@ -21,4 +21,4 @@ managed instance group.
 |------|-------------|
 | self\_link | Self-link of the managed instance group |
 
-[^]: (autogen_docs_end)
\ No newline at end of file
+[^]: (autogen_docs_end)
```

Comment:
this is the old style doc generator marker, please replace it across the code base using [this function](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/concourse/build/developer-tools/build/scripts/task_helper_functions.sh#L186)

Reply:
fixed